esp-hal-common/Cargo.toml

```
@@ -25,6 +25,11 @@ procmacros       = { version = "0.1.0", package = "esp-hal-procmacros", path = "
 void             = { version = "1.0.2", default-features = false }
 embedded-dma     = "0.2.0"
 
+# async
+embassy-sync       = { package = "embassy-sync", git = "https://github.com/embassy-rs/embassy/", rev = "92ed957", optional = true }
+embassy-time       = { package = "embassy-time", git = "https://github.com/embassy-rs/embassy/", rev = "92ed957", optional = true, features = ["nightly"] }
+embedded-hal-async = { version = "0.1.0-alpha.1", optional = true }
```

```suggestion
embedded-hal-async = { version = "0.1.0-alpha.3", optional = true }
```

There have been a couple updates released, we should try to stay as up to date as possible.